packages/desktop/electron/main.js

```diff
@@ -423,6 +424,15 @@ ipcMain.handle('check-if-file-exists', (_e, filePath) => {
     return fs.existsSync(`${directory}/__storage__/${filePath}`)
 })
 
+function ensureDirectoryExistence(filePath) {
```

Comment:
Perhaps if we are not using this in the PR we can add it as a PR onto develop if it is an improvement we think its worth having

Reply:
Nice idea, added here: https://github.com/iotaledger/firefly/pull/6652/files